src/main/java/com/revature/controller/JobController.java

```diff
@@ -73,8 +76,34 @@ public JobController(JobService jobService, SkillService skillService) {
 	 the page to search for jobs.
 	 * @param page An integer identifying the page to search for jobs.
 	 * @return A list of all jobs on the specified page. */
-    public List<Job> getAllPaged(@PathVariable int page){
-    	return jobService.getAllJobsPaged(page);
+    public List<Job> getAllPaged(@PathVariable int page, @RequestHeader("usefilter") boolean useFilter,
```

Comment:
These should not be RequestHeaders. These should be QueryParams (the stuff after '?' in the URL). This allows you to remove 'useFilter' because in code you can check whether that query parameter was available.

Reply:
fixed